extensions/trace_propagators/src/main/java/io/opentelemetry/extensions/trace/propagation/Common.java

```diff
@@ -52,8 +52,8 @@ static SpanContext buildSpanContext(String traceId, String spanId, String sample
               : NOT_SAMPLED_FLAGS;
 
       return SpanContext.createFromRemoteParent(
-          TraceId.fromLowerBase16(StringUtils.padLeft(traceId, MAX_TRACE_ID_LENGTH), 0),
-          SpanId.fromLowerBase16(spanId, 0),
+          StringUtils.padLeft(traceId, MAX_TRACE_ID_LENGTH),
+          spanId,
```

Comment:
We can replace some of the methods in this class for checking validity I think

Reply:
You mean use the ones on TraceId/SpanId?

Reply:
Yeah

Reply:
done